parm/soca/jcb-base.yaml.j2

```diff
@@ -75,10 +75,10 @@ marine_obsdatastats_path: '{{COMOUT_OCEAN_ANALYSIS}}/diags'
 
 marine_obsdatain_path: '{{DATA}}/obs'
 marine_obsdatain_prefix: '{{OPREFIX}}'
-marine_obsdatain_suffix: '.{{ current_cycle | to_YMDH }}.nc4'
+marine_obsdatain_suffix: '.nc'
```

Comment:
In this suffix used for creating an obs file (whether pre or post IODA-conversion) in the RUNDIR? The cycle gets added to the filename there to manage multi-cycle windows.

Reply:
The multi-cycle windows is implemented in obsForge now @AndrewEichmann-NOAA .